data/rbac-schema/src/main/kotlin/net/corda/rbac/schema/RbacKeys.kt

```diff
@@ -15,4 +15,35 @@ object RbacKeys {
      * Separator for permission string which sits between prefix and the wildcard expression
      */
     const val PREFIX_SEPARATOR = ":"
+
+    /**
+     * Regular expressions to validate common data structures as part of RBAC checks
+     */
+    private const val UUID_CHARS = "[a-fA-F0-9]"
+    const val UUID_REGEX = "$UUID_CHARS{8}-$UUID_CHARS{4}-$UUID_CHARS{4}-$UUID_CHARS{4}-$UUID_CHARS{12}"
+
+    /**
+     * vNode short hash
+     */
+    const val VNODE_SHORT_HASH_REGEX = "$UUID_CHARS{12}"
+
+    /**
+     * RBAC user name regex
+     */
+    const val USER_REGEX = "[-._@a-zA-Z0-9]{3,255}"
```

Comment:
Regex's for valid user names are very difficult. There are lots of people with unusual characters in their name e.g. ' or French names with accents. I think we need to consider alternative escaping strategies here

Reply:
I have raised a follow-up Jira to that effect: https://r3-cev.atlassian.net/browse/CORE-7305